common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationRequest.java

```diff
@@ -293,5 +255,12 @@ private void addPkceChallengeToRequestParameters(@NonNull final Map<String, Stri
         // If excluded, code_challenge is assumed to be plaintext if code_challenge is included.
         // Azure AAD v2.0 supports both plain and S256.
         requestParameters.put(CODE_CHALLENGE_METHOD, getPkceChallenge().getCodeChallengeMethod());
+
+    }
+
+    @Override
+    public String getAuthorizationEndpoint(){
+        //TODO: Need to take authority aliasing via instance discovery into account here
+        return "https://login.microsoftonline.com/common/oauth2/v2.0/authorize";
```

Comment:
It seems the AuthorizationEndpoint is not a property of request anymore. I'm confused on the getter here. Where should it set the AuthorizationEndpoint value? Are we going to move the instance discovery functions into the getAuthorizationEndpoint()?

Reply:
Let's talk about this.  Instance discovery should be part of AAD... not part of a request.